client/cli/src/params/network_params.rs

```diff
@@ -41,6 +41,11 @@ pub struct NetworkParams {
 	#[structopt(long = "reserved-only")]
 	pub reserved_only: bool,
 
+	/// The public address that other nodes will use to connect to it.
+	/// This can be used if there's a proxy in front of this node.
+	#[structopt(long = "public-addr", value_name = "PUBLIC_ADDR")]
+	pub public_addr: Vec<Multiaddr>,
```

Comment:
```suggestion
	#[structopt(long, value_name = "PUBLIC_ADDR")]
	pub public_addr: Vec<Multiaddr>,
```

The value is automatically generated by structopt.

Reply:
This is inconsistent with the rest of the file, but I guess the rest of the file can be updated in a later PR.

Reply:
Yes we just update it little by little. It's really minor after all